build.gradle.kts

```diff
@@ -1,32 +1,32 @@
-
 import io.papermc.hangarpublishplugin.model.Platforms
 import net.minecrell.pluginyml.bukkit.BukkitPluginDescription
 import xyz.jpenilla.runpaper.task.RunServer
+import kotlin.system.exitProcess
 
 plugins {
-    java
-    `java-library`
+    id("idea")
+    id("java")
+    id("java-library")
     id("olf.build-logic")
+
     id("com.diffplug.spotless") version "6.18.0"
+    id("com.modrinth.minotaur") version "2.+"
     id("io.github.goooler.shadow") version "8.1.7"
+    id("io.papermc.hangar-publish-plugin") version "0.1.2"
     id("net.minecrell.plugin-yml.bukkit") version "0.5.3"
     id("xyz.jpenilla.run-paper") version "2.1.0"
```

Comment:
Use Version Catalog: https://docs.gradle.org/current/userguide/platforms.html#sec:sharing-catalogs

Reply:
I did some tinkering and digging and I think it is way too overkill for such a small project with such few dependencies

Reply:
In fact, a version catalog wouldn't work for plugins as described here https://github.com/gradle/gradle/issues/24876

Reply:
It would make a lot of sense, plus it works with the plugins.
Look here for the plugins: https://github.com/IntellectualSites/ClipboardConnect/blob/main/build.gradle.kts#L9-L14

Reply:
done